.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+version: 2.1
+
+jobs:
+  jdk8:
+    docker:
+      - image: circleci/openjdk:8-stretch
+    steps:
+      - checkout
+      - run: mvn verify -V
+
+  jdk11:
+    docker:
+      - image: circleci/openjdk:11-stretch
+    steps:
+      - checkout
+      - run: mvn verify -V
+
+  jdk15:
+    docker:
+      - image: circleci/openjdk:15-buster
+    steps:
+      - checkout
+      - run: mvn verify -V
+
+  sonar:
+    docker:
+      - image: circleci/openjdk:8-stretch
+    steps:
+      - checkout
+      - run: > 
+          mvn verify
+          -P !build-extras
+          -Dtest="*,!RunnerStandardIOTest,!RunnerWebSocketTest"
```

Comment:
as a reminder, this parameter can be strange but it is because jacoco is not able to calculate coverage correctly with those tests (most of the time).